### ISSUE TITLE
Tap event fired twice on Windows Tablet

### ISSUE THREAD

**Issue by andyforever** [madrobby/zepto#1127]
Code

``` javascript
<!DOCTYPE html>
<html>
  <head>
    <meta name="viewport" content="width=device-width,initial-scale=1.0,user-scalable=no" />
    <script src="./lib/zepto.min.js"></script> 
    <script src="./lib/touch.js"></script> 
    <style type="text/css">
      body {
        -ms-touch-action: none !important;
      }
    </style>
  </head>   
  <body>
    <input type="text" id="in" ></input>
    <button id="btn1" style="width:100px;height:100px">a</button>
    <button id="btn2" style="width:100px;height:100px">b</button>

    <script>
      Zepto(function($) {
          var value = '';

          $('#btn1').tap( function(event) {
              value = $('#in').val() + 'a';
              $('#in').val(value);
              return false;
          });

          $('#btn2').tap( function(event) {
              value = $('#in').val() + 'b';
              $('#in').val(value);
              return false;
          });
      })  
    </script>
  </body>
</html>
```

**Comment by madrobby** [comment]
Hi there! Does this still happen? What OS & browser version do you use exactly?

**Comment by daizch** [comment]
@madrobby Hi, I met this problem too.

OS: Mac OS X
browser: Chrome/53.0.2785.116

Here is my code

```html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width,initial-scale=1.0,minimum-scale=1.0,maximum-scale=1.0,user-scalable=no">
    <meta content="yes" name="apple-mobile-web-app-capable">
    <meta content="black" name="apple-mobile-web-app-status-bar-style">
    <meta content="telephone=no" name="format-detection">
    <meta content="email=no" name="format-detection">
    <script src="./lib/zepto.js"></script>
    <script src="./lib/touch.js"></script>
    <title>demo</title>
</head>
<body>
<div class="btn">tap me</div>

<script>
    window.onload = function () {
        $(document.body).on('tap', '.btn', function () {
            console.log('tap');
        });
    };
</script>
</body>
</html>
```

**Comment by madrobby** [comment]
This should be fixed in d7e4cafd129cdb6223e6c3c60c7f6ccfc6750512. Please reopen this issue if you still have issues on the latest `master`.
